blog/views.py

```diff
@@ -4,58 +4,56 @@
 from user.models import Users
 from django.http import HttpResponse, HttpResponseRedirect
 from django.contrib.auth.models import User
-import random
-import string
 
 # Create your views here.
 def create(request):
     if request.method == "POST":
         form = (request.POST)
-        check = Users.objects.get(user_id=request.user.id)
-        article = Articles(user = check, title= form["title"], body = form["body"], status = form['status'])
+        user = Users.objects.get(user_id=request.user.id)
+        article = Articles(user = user, title= form["title"], body = form["body"], status = form['status'])
         article.save()
-        tags = form["tags"].split(",")
-        if tags[0] != '':
-            for i in tags:
-                query_set = Tags.objects.filter(tag_name= i )
+        if len(form["tags"]) > 0:
+            tags = form["tags"].split(",")
+            for tag in tags:
+                query_set = Tags.objects.filter(tag_name= tag )
                 if query_set.exists():
                     mapping = ArticleTagMapping(article_id = article.id, tag_id = query_set[0].id)
                 else:
-                    tag= Tags(tag_name = i)
+                    tag= Tags(tag_name = tag)
                     tag.save()
                     mapping = ArticleTagMapping(article_id = article.id, tag_id = tag.id)
                 mapping.save()
         messages.info(request,"New article has been created with title {}".format(form["title"]))
-        return HttpResponseRedirect('home')
+        return HttpResponseRedirect("/create")
     else:
         return(render(request,
-                "createArticle.html",
-                context={"check":"check"})
+                "createArticle.html")
             )
 
-def view(request, title,slug):
-    article = Articles.objects.filter(title = title, slug = slug)
-    tags = ArticleTagMapping.objects.filter(article_id = article[0].id)
```

Comment:
You don not need ArticleTagMapping table.
https://docs.djangoproject.com/en/3.0/topics/db/examples/many_to_many/
But for now it is Ok.